automation/cli.py

```diff
@@ -62,16 +75,31 @@ def hex_bytes(value):
 location_config_parser.add_argument("-lat", "--latitude", type=float, help="Latitude in degrees")
 location_config_parser.add_argument("-lng", "--longitude", type=float, help="Longitude in degrees")
 location_config_parser.add_argument("-ele", "--elevation", type=float, help="Elevation in meters")
-prediction_config_parser = config_subparsers.add_parser("global", help="Change global prediction parameters")
-prediction_config_parser.add_argument("-aos", type=int, help="Elevation (in degress) on AOS")
-prediction_config_parser.add_argument("-me", "--max-elevation", type=int, help="Max elevation greater than")
-prediction_config_parser.add_argument("-s", "--strategy", choices=["aos", "max-elevation"], help="Select strategy to track satellites")
+global_config_parser = config_subparsers.add_parser("global", help="Change global prediction parameters")
+global_config_parser.add_argument("-aos", type=int, help="Elevation (in degress) on AOS")
```

Comment:
We need some docs to explain how to use those parameter. The built-in help is useless. Or at least I'm too dumb to force it to show me all the options. I finally figured out how to set the longitude (incorrectly set in the default config to be 54.9), but it was trial-and-error with looking at the sources.

Reply:
I added issue #41 for prepare README file.